HelloWorldMetal/trace_model.py

```diff
@@ -0,0 +1,9 @@
+import torch
```

Comment:
Can we drop this file for now? After we get the MacOS nightly working, we can check in this script.